docker/Dockerfile-release

```diff
@@ -15,7 +15,7 @@ exec java \$JAVA_OPTS -jar \$APP_FILE' > /start.sh && chmod +x /start.sh"
 
 VOLUME /tmp
 
-RUN wget -O /$APP_FILE $APP_DOWNLOAD_URL
+RUN wget -O /$APP_FILE $APP_DOWNLOAD_URL && mkdir /plugins && chmod a+w /var /plugins
```

Comment:
Two questions:
1. You create and apply permissions on `plugins` folder because plugins uploads in this folder?
2. You apply permissions on `/var` due to file logging?

Reply:
1/ correct
2/ correct

also might be that /data folder is required and needs permissions

Reply:
basically good test is to add step into dockerfile

USER nobody 

and try to start

Reply:
1/ correct
2/ correct

Also directory /data might be required with permissions on it

Basically good test is to add into docker file line

USER nobody 

and try to start the application properly

Reply:
1. I believe we already moved all plugin uploads to Minio (probably in current development version, 5.1), and there shouldn't be any other option to upload plugins straight on disc in cluster deployments. Cloud applications shouldn't consider itself as a storage.
2. Then we need to disable any log output, the same as for p.1. I created internal ticket for this.

The same for `/data` directory. It should be mounted as an external volume with permissions applied on Kubernetes deployment chart level.

Reply:
One option would be to add persistanceVolumeClaim as template:

apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: api-service-data
spec:
  storageClassName: nfs
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: {{ $.Values.reportportal.data.persistentVolumeClaimSize }}

And then mount it to locations
 volumeMounts:
            - mountPath: /data
              name: api-service-data

and define volumes as:
volumes:
        - name: api-service-data
          persistentVolumeClaim:
            claimName: api-service-data

Could work like that..

Reply:
We removed any logging into files by default. And your suggestion for plugins is for kubernetes deployment: https://github.com/reportportal/kubernetes

Reply:
is that now integrated for next release 5.1?
also this https://github.com/reportportal/migrations/pull/140 ?

thanks

Reply:
No need update for /var dir with latest 5.1 as tested.
For /plugins dir even though data will be stored via minio, still need the group write access to the dir, may update with 'g+w' rather than 'a+w'.